docroot/sites/all/modules/custom/midcamp_eventbrite/plugins/content_types/event_info.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Plugins are described by creating a $plugin array which will be used
+ * by the system that includes this file.
+ */
+$plugin = array(
+  'single' => TRUE,
+  'title' => t('Event info'),
+  'description' => t('Displays event info from Eventbrite'),
+  'category' => t('Eventbrite'),
+);
+/**
+ * Renders the form.
+ *
+ * @param $subtype
+ * @param $conf
+ * @param $panel_args
+ * @param $context
+ * @return \stdClass
+ */
+function midcamp_eventbrite_event_info_content_type_render($subtype, $conf, $panel_args, &$context) {
+  $block = (object) array(
+    'subject' => '',
+    'content' => array(
+      '#markup' => '@todo embed iframe',
```

Comment:
When event pushed live, can put iframe code here, or whatever the embed thing is.